rest/src/main/java/org/jboss/pnc/rest/notifications/model/NotificationFactory.java

```diff
@@ -0,0 +1,34 @@
+package org.jboss.pnc.rest.notifications.model;
+
+import org.jboss.pnc.spi.BuildStatus;
+import org.jboss.pnc.spi.events.BuildStatusChangedEvent;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.util.HashMap;
+import java.util.Map;
+
+@ApplicationScoped
+public class NotificationFactory {
+
+    private Map<BuildStatus, NotificationEventType> externalEvents = new HashMap<>();
+
+    public NotificationFactory() {
+        externalEvents.put(BuildStatus.REPO_SETTING_UP, NotificationEventType.BUILD_STARTED);
```

Comment:
I find it more confusing having another status for the same thing. We already have BuildStatus, BuildSetStatus and BuildStatusChangedEvent, BuildSetStatusChangedEvent as part of SPI and already used in the REST.
https://github.com/project-ncl/pnc/blob/33ebb8c8d7dd2594f878ac1f5d1ed0cdb3c73009/rest/src/main/java/org/jboss/pnc/rest/trigger/BuildTriggerer.java#L102